apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestDataPathsActivity.java

```diff
@@ -538,13 +573,15 @@ void testOutputDeviceCombo(int deviceId,
             int result = testResult.result;
             if (result == TEST_RESULT_FAILED) {
                 if (deviceType == AudioDeviceInfo.TYPE_BUILTIN_EARPIECE
-                        && numOutputChannels == 2
-                        && outputChannel == 1) {
+                        && outputChannels == 2
```

Comment:
What if this contains a mask?

Reply:
I changed the function to always take in a count. We can just check that